anndata/_core/sparse_dataset.py

```diff
@@ -333,6 +333,19 @@ def _get_group_format(group) -> str:
         return _read_attr(group.attrs, "encoding-type").replace("_matrix", "")
 
 
+# Check for the overridden few methods above in our BackedSparseMatrix subclasses
+def is_sparse_indexing_overridden(format, row, col):
+    format_idx = int(format == "csc")
+    indexer = [row, col]
+    major_indexer = indexer[format_idx]
+    minor_indexer = indexer[format_idx - 1]
```

Comment:
cute, but I think this would be cleaner:

```suggestion
    major_indexer, minor_indexer = (row, col) if format == "csr" else (col, row)
```